### DISCUSSION TITLE
Curriculum learning for DirectRL workflow

### DISCUSSION THREAD

**Discussion by Oussama1209** [isaac-sim/IsaacLab#1533] [Q&A]
I want to implement a curriculum learning inside a directrl workflow, and based on the iteration number change the strategy for the learning, but I can't access directly the value of current iteration number. What I need to do is access it from the directory of IsaacLab installation inside on_policy_runner to get the value of current iteration.
I'm sure there is a better way to do it if anyone knows !

**Comment by RandomOakForest** [comment]
Thank you for posting this question. This seems to be covered in the doc for [creating a manager based RL environment](https://isaac-sim.github.io/IsaacLab/main/source/tutorials/03_envs/create_manager_rl_env.html#defining-curriculum), though it can also be implemented in a direct RL workflow. Let us know if you need further help.
